plugins/rts54hub/rts54hub.quirk

```diff
@@ -0,0 +1,5 @@
+# RTS5423 Development Board
+[DeviceInstanceId=USB\VID_0BDA&PID_5423]
+Plugin = rts54hub
+FirmwareSizeMin = 0x20000
+FirmwareSizeMax = 0x20000
```

Comment:
From other issue discussion on VID/PID and that it's project specific, not everyone changes VID/PID I think it's worth bumping this up to `0x40000` to be safe.

Reply:
WFM.